### ISSUE TITLE
Queueing and playing folders in filemode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peteaky/android-xbmcremote#114]
```
In the XBMC gui i can press "play" (Not sure what exact command is mapped) 
and I get a menu with settings, and queue etc. If I browse music in 
xbmcremote I can't actually enqueue or play/queue an entire folder.
```

Original issue reported on code.google.com by `sop...@gmail.com` on 30 Nov 2009 at 2:30

**Comment by GoogleCodeExporter** [comment]
```
Did you scan your music in XBMC? You can play and queue albums, songs by artist 
or 
genre. That's what library mode is for.
```

Original comment by `phree...@gmail.com` on 1 Dec 2009 at 2:07
- Changed state: **Accepted**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I did, but I always use this feature in XBMC itself. Yes, my music is badly 
sorted/tagged so library mode is less useful for me. 

Anyway, its a feature that should not disrupt anything else.
```

Original comment by `sop...@gmail.com` on 1 Dec 2009 at 2:41

**Comment by GoogleCodeExporter** [comment]
```
I would be really good to have it playing the next song if I start a song in 
filemode.
Everybody don't use librarymode.

Keep up the good work.
```

Original comment by `Henrik.J...@gmail.com` on 20 Dec 2009 at 8:08

**Comment by GoogleCodeExporter** [comment]
```
Issue 140 has been merged into this issue.
```

Original comment by `till.ess...@googlemail.com` on 3 Jan 2010 at 1:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `phree...@gmail.com` on 5 Mar 2010 at 4:21
- Added labels: **Priority-Medium**
- Removed labels: **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
```
Issue 137 has been merged into this issue.
```

Original comment by `phree...@gmail.com` on 5 Mar 2010 at 4:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `phree...@gmail.com` on 5 Mar 2010 at 4:56
- Changed title: **Queueing and playing folders in filemode**

**Comment by GoogleCodeExporter** [comment]
```
i can also confirm, in filemode "Queue Folder" and "Play Folder" doesn´t work. 
And a 
very nice feature would be if you play a track in filemode it just plays the 
next 
song instead of not playing anything. But know the same problem our made by 
design 
in the XBMC Web Interface control.
```

Original comment by `danny.ni...@gmail.com` on 10 Apr 2010 at 6:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `phree...@gmail.com` on 21 Apr 2010 at 8:21
- Added labels: **Milestone-Release0.8**

**Comment by GoogleCodeExporter** [comment]
Original comment by `phree...@gmail.com` on 21 Apr 2010 at 8:22
- Added labels: **Milestone-Release1.0**
- Removed labels: **Milestone-Release0.8**

**Comment by GoogleCodeExporter** [comment]
```
Issue 215 has been merged into this issue.
```

Original comment by `phree...@gmail.com` on 21 Apr 2010 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
I put a vote on this one - it is afaik the only way to play last.fm streams. . 
```

Original comment by `msiegumf...@gmail.com` on 23 Apr 2010 at 3:52

**Comment by GoogleCodeExporter** [comment]
```
Bumped.
```

Original comment by `phree...@gmail.com` on 24 May 2010 at 3:18
- Added labels: **Milestone-Release0.8**, **Priority-High**
- Removed labels: **Milestone-Release1.0**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
For those eager to test, you should now be able to "play" folders via music 
filemode 
(since r642). Queue will follow shortly.

However if you have a playlist in your folder, it will be added as well.
```

Original comment by `phree...@gmail.com` on 24 May 2010 at 10:25
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r642 and r643 and upcoming v0.7.1. Post here if you still got problems.
```

Original comment by `phree...@gmail.com` on 25 May 2010 at 7:38
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
i just testet 0.7.1 and it seems very stable an works good with playing 
folders. 
Tested it on OSX Leopard with XBMC 9.11. still some nice to have features

what i had problem with was its not possible to say play folder form inside the 
mp3 
folder. it only works if you say play folder, on a folder. don´t know if its 
by 
design.

another thing that could be nice is if it plays the next song just if your 
inside a 
album folder and plays one song from here. Now it only ques the song you pressed
```

Original comment by `danny.ni...@gmail.com` on 26 May 2010 at 8:36
